### PR TITLE
Release v.4.0.3

### PR DESCRIPTION
**Pull request by mek-x**

### CODE REVIEW

**Comment by pietrushnic** [comment]
@mek-x same problems with emails in this PR.

**Comment by mek-x** [comment]
@pietrushnic now it should be ok

File: payloads/external/SeaBIOS/Makefile.inc [hunk 1]
```diff
@@ -1,5 +1,5 @@
 SEABIOSGIT-$(CONFIG_SEABIOS_GENERIC)=http://review.coreboot.org/p/seabios.git
-SEABIOSGIT-$(CONFIG_SEABIOS_ELTAN)=https://github.com/pcengines/seabios.git
+SEABIOSGIT-$(CONFIG_SEABIOS_ELTAN)=git@81.95.197.197:pcengines/seabios.git
```
Comment (pietrushnic) [inline payloads/external/SeaBIOS/Makefile.inc]: @mek-x you probably missed that.

Comment (mek-x) [inline payloads/external/SeaBIOS/Makefile.inc]: yes... fixing

File: payloads/external/SeaBIOS/Makefile.inc [hunk 2]
```diff
@@ -70,7 +70,7 @@ ifeq ($(CONFIG_SEABIOS_VGA_COREBOOT),y)
 	echo "CONFIG_BUILD_VGABIOS=y" >> seabios/.config
 endif
 
-	echo "CONFIG_DEBUG_LEVEL=0" >> seabios/.config
+	echo "CONFIG_DEBUG_LEVEL=1" >> seabios/.config
```
Comment (mek-x) [inline payloads/external/SeaBIOS/Makefile.inc]: CONFIG_DEBUG_LEVEL has to be at least 1, even when using dprintf level 0 or output is disabled.

**Pull request merged**